src/nr_openai_observability/langchain_callback.py

```diff
@@ -187,6 +195,20 @@ def on_agent_action(self, action: AgentAction, **kwargs: Any) -> Any:
 
     def on_agent_finish(self, finish: AgentFinish, **kwargs: Any) -> Any:
         """Run on agent end."""
+        final_message = {
```

Comment:
What is this particular call trying to accomplish? I see it is creating a message, but not sure how it is supposed to relate back to the rest of the response.

Reply:
This was based on a thread before I left, but looking back on it I realize we never settled on if this was a good idea. Essentially this creates an LLM message event related by transaction and trace ID containing the actual text resturned to the user by langchain with the user provided message_id. 

We can probably capture at least some extra data like we are the model to get the completion ID it's associated with as well.

Reply:
This has been removed from this PR, langchain feedback logic will be included in a later PR